install/resources/mybb_theme.xml

```diff
@@ -822,6 +822,14 @@ blockquote cite span.highlight {
 	font-size: 13px;
 }
 
+.mycode_me {
```

Comment:
The master file is still updated. IMHO the best solution would be one which works without the class (in case the file has been modified and isn't updated) and with the class (in case nothing has been modified and the master file is used). So if someone wants to use a different color he'll be able to add the class and change the color.